android/src/main/kotlin/io/customer/customer_io/CustomerIoPlugin.kt

```diff
@@ -153,6 +151,9 @@ class CustomerIoPlugin : FlutterPlugin, MethodCallHandler {
         val organizationId = configData.getProperty<String>(
             Keys.Environment.ORGANIZATION_ID
         )?.takeIfNotBlank()
```

Comment:
I would suggest to completely remove `organizationId` from native files and rely on `enableInApp` only, so it is convenient to generate warnings for desired platform and do the migrations for breaking changes later. I followed the [same idea in react native](https://github.com/customerio/customerio-reactnative/blob/8b97125d471f6a332722cee7963f1e48de29ef5b/src/CustomerioTracking.tsx#L54-L58) as well.